cli/cmd/contracts.go

```diff
@@ -94,6 +95,34 @@ var (
 		},
 	}
 
+	contractsDumpCodeCmd = &cobra.Command{
+		Use:   "dump-code <code-id>",
+		Short: "Dumps the contract code in Base64 format",
+		Args:  cobra.ExactArgs(1),
+		Run: func(cmd *cobra.Command, args []string) {
+			cfg := cliConfig.Global()
+			npa := common.GetNPASelection(cfg)
+			strCodeID := args[0]
+
+			if npa.ParaTime == nil {
+				cobra.CheckErr("no paratimes configured")
+			}
+
+			codeID, err := strconv.ParseUint(strCodeID, 10, 64)
+			cobra.CheckErr(err)
+
+			ctx := context.Background()
+			conn, err := connection.Connect(ctx, npa.Network)
+			cobra.CheckErr(err)
+
+			// Fetch WASM contract code, if supported.
+			codeStorage, err := conn.Runtime(npa.ParaTime).Contracts.CodeStorage(ctx, client.RoundLatest, contracts.CodeID(codeID))
+			cobra.CheckErr(err)
+
+			fmt.Print(base64.StdEncoding.EncodeToString(codeStorage.Value))
```

Comment:
Why encode the output in base64, is that a more common use case?

Reply:
Not sure. Changed to binary to simplify chaining other commands.